qemu/tests/cfg/guest_iommu_test.cfg

```diff
@@ -10,6 +10,8 @@
     intel_iommu = yes
     virtio_dev_iommu_platform = on
     virtio_dev_ats = on
+    virtio_dev_disable_legacy = on
+    virtio_dev_disable_modern = false
```

Comment:
hi @XueqiangWei , 
as you said this afternoon, we cannot hit the issue when driveformat=virtio_scsi(no pci bridge), so here do you think we add these two line under virtio_blk, i.e.
virtio_blk:
       virtio_dev_disable_legacy = on
       virtio_dev_disable_modern = false
Then this setting will not affect virtio_scsi, just leave virtio_scsi as it is

Reply:
Yes, make sense. I will update it. Thanks.